src/boot/efi/meson.build

```diff
@@ -102,6 +102,46 @@ if have_gnu_efi
         efi_conf.set10('ENABLE_TPM', get_option('tpm'))
         efi_conf.set('SD_TPM_PCR', get_option('tpm-pcrindex'))
 
+        if get_option('sbat_distro') != ''
+                efi_conf.set_quoted('SBAT_PROJECT', meson.project_name())
+                efi_conf.set_quoted('PROJECT_VERSION', substs.get('PROJECT_VERSION'))
+                efi_conf.set_quoted('PROJECT_URL', substs.get('PROJECT_URL'))
+                efi_conf.set_quoted('SBAT_DISTRO_GENERATION', '@0@'.format(get_option('sbat_distro_generation')))
```

Comment:
If the option is an integer, shouldn't this just be saved as `efi_conf.set('SBAT_DISTRO_GENERATION', get_option('sbat_distro_generation')` ?

Reply:
i can do that, but then also will need to tweak the csv header.